roles/nginx/tasks/install.yml

```diff
@@ -8,6 +8,12 @@
     - { path: "{{ nginx_conf_dir }}/usb-lib.conf" }
     - { path: "{{ nginx_conf_dir }}/modules.conf" }
 
+- name: Stop '{{ apache_service }}' systemd service
+  systemd:
+    name: "{{ apache_service }}"    # apache2 or httpd, per /opt/iiab/iiab/vars/<OS>.yml
+    state: stopped
+  ignore_errors: yes
+
```

Comment:
That is why httpd/tasks/install.yml had(before you disabled) the final systemd call set to stopped in that code block. Things were a certain way for a reason, to avoid using 'ignore_errors' needlessly. Don't forget to use apache_installed if you want to avoid ignore_errors red warning when apache may not be installed in the future.